cmd/optimize/events.go

```diff
@@ -82,6 +83,12 @@ type recommendationRow struct {
 	Change             string
 }
 
+type NoEventsError struct{}
+
+func (e *NoEventsError) Error() string {
```

Comment:
```suggestion
func (e *ErrNoEvents) Error() string {
```
To comply with error type naming conventions